.github/workflows/githubRelease.yml

```
@@ -7,14 +7,17 @@ on:
         required: false
         default: false
         description: use a prerelease instead of a regular release
+    secrets:
+      ALT_GITHUB_TOKEN:
+        required: false
```

Not sure if you can do this but it would simplify the assignments down below.
If this isn't possible maybe we could pass the secret in as an input or env var?

```suggestion
    secrets:
      ALT_GITHUB_TOKEN:
        required: false
        default: secrets.SVC_CLI_BOT_GITHUB_TOKEN
```